egui/src/input_state.rs

```diff
@@ -192,6 +192,23 @@ impl InputState {
         self.pointer.wants_repaint() || self.scroll_delta != Vec2::ZERO || !self.events.is_empty()
     }
 
+    // Ignore a key if it was pressed or released this frame. Useful for hotkeys.
+    // Returns if the key was pressed this frame
+    pub fn ignore_key(&mut self, ignore_key: Key, modifiers: Modifiers) -> bool {
```

Comment:
I think `consume_key` might be a better name

```suggestion
    /// Check for a key press. If found, `true` is returned and the key pressed i consumed, so that this will only return `true` once.
    pub fn consume_key(&mut self, modifiers: Modifiers, ignore_key: Key) -> bool {
```

Reply:
Note also the docstring with `///`